server/src/v1/controllers/sessionsController.js

```diff
@@ -66,20 +66,20 @@ class SessionsController {
     const { sessionId } = req.params;
 
     try {
-      const {
-        mentorId, menteeId, questions, menteeEmail, status,
-      } = Session.accept(sessionId);
-
+      const rows = await Session.accept(sessionId);
+      const mentees = await User.findByEmail(rows[0].mentee_email)
       return res.status(200).json({
         status: 200,
         message: 'SUCCESS',
         data: {
-          sessionId,
-          mentorId,
-          menteeId,
-          questions,
-          menteeEmail,
-          status,
+          data: {
```

Comment:
Why are you nesting data?

Reply:
Had not realized, will correct and re-push to remote.

Reply:
Resolved